starter-code/src/app/components/my-movie/my-movie.component.html

```diff
@@ -0,0 +1,22 @@
+<p>
+  my-movie works!
+</p>
+
+<div>
+    <div>
+      <h3>Room {{movie.room}}</h3>
```

Comment:
Now we know that at the beginning `movie` it will be undefined. Don't forget to handle this error with a `*ngIf`